src/ngraph/frontend/onnx_import/op/shrink.cpp

```diff
@@ -43,14 +43,24 @@ namespace ngraph
                     ASSERT_VALID_ARGUMENT(node, !(lambd < 0.0f))
                         << " The provided 'lambd' value:" << lambd << " must not be negative.";
 
-                    const auto negative_lambd = ngraph::op::Constant::create(
-                        input->get_element_type(), input->get_shape(), {-lambd});
+                    std::shared_ptr<ngraph::op::Constant> negative_lambd;
+                    const auto input_element_type = input->get_element_type();
+                    if (input_element_type.is_signed())
+                    {
+                        negative_lambd = ngraph::op::Constant::create(
+                            input_element_type, input->get_shape(), {-lambd});
+                    }
+                    else
+                    {
+                        negative_lambd = ngraph::op::Constant::create(
```

Comment:
Please add some comment explaining why `negative_lambd` is actually equal to zero.